src/partials/customer-reviews.html

```diff
@@ -1,11 +1,13 @@
-<section class="customer-reviews"></section>
-<div class="container">
-  <img src="images/review-photo.jpg" alt="" class="customer-reviews__photo" />
-  <div class="customer-reviews__column">
-    <p class="customers-reviews__text">
-      Nullam viverra consectetuer. Quisque cursus et, porttitor risus. Aliquam sem. In hendrerit
-      nulla quam nunc, accumsan congue. Lorem ipsum primis in nibh vel risus.
-    </p>
+<section class="customer-reviews" id="About us">
+  <div class="container">
+    <img src="images/review-photo.jpg" alt="" class="customer-reviews__photo" />
+    <div class="customer-reviews__column">
+      <p class="customers-reviews__text">
+        Nullam viverra consectetuer. Quisque cursus et, porttitor risus. Aliquam sem. In hendrerit
+        nulla quam nunc, accumsan congue. Lorem ipsum primis in nibh vel risus.
+      </p>
+      </div>
```

Comment:
niepotrzebne to zamknięcie `</div>` - powoduje wypadnięcie `<p class="customer-reviews__sign">Emily, Los Angeles</p>` poza container